src/mero_conf.erl

```diff
@@ -217,3 +223,33 @@ get_env(Key) ->
         undefined ->
             exit({undefined_configuration, Key})
     end.
+
+
+process_value({servers, {elasticache, ConfigEndpoint, ConfigPort}}) ->
+    process_value({servers, {elasticache, [{ConfigEndpoint, ConfigPort, 1}]}});
+process_value({servers, {elasticache, ConfigList}}) when is_list(ConfigList) ->
+    HostsPorts = lists:foldr(
```

Comment:
What do you think about reducing this huge `foldr` to something like the following (not tested on a console, just a concept) code?
```erlang
    HostsPorts = [Port || HostConfig <- ConfigList, Port <- get_ports(HostConfig)].
```
…and then somewhere else…
```erlang
get_ports({HostIn, PortIn}) ->
    get_ports({HostIn, PortIn, ClusterSpeedFactorIn});
get_ports({HostIn, PortIn, ClusterSpeedFactorIn}) ->
    lists:duplicate(ClusterSpeedFactor, get_elasticache_cluster_config(Host, Port)).
```

(you probably have to choose better names for this)